analyzers/tests/SonarAnalyzer.Test/Wrappers/RegisterSymbolStartActionWrapperTest.cs

```diff
@@ -18,6 +18,7 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
+using Microsoft.CodeAnalysis;
```

Comment:
this seems to be unused

Reply:
Needed for `[ExcludeFromCodeCoverage]`

Reply:
Hmm, that's strange. I've removed it on my local env and I can build just fine. Also, that attribute is not used in this file.

Reply:
I looked in the wrong file. -> Removed.